README.md

```diff
@@ -1,7 +1,7 @@
 <img src=http://storage.mds.yandex.net/get-devtools-opensource/250854/catboost-logo.png width=300/>
 
-[Documentation](https://tech.yandex.com/catboost/doc/dg/concepts/about-docpage/) |
-[Installation](https://tech.yandex.com/catboost/doc/dg/concepts/cli-installation-docpage/)
+[Documentation](https://tech.yandex.com/catboost/doc/dg/concepts/about-docpage/) | [Python Installation](https://tech.yandex.com/catboost/doc/dg/concepts/python-installation-docpage/) |
+[R Installation](https://tech.yandex.com/catboost/doc/dg/concepts/cli-installation-docpage/)
```

Comment:
I guess they just fixed the link. 

Nevertheless, I would prefer to see clear links to python and R installations.